src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -308,5 +316,18 @@ void VRModule::UnregisterRequest(uint32_t correlation_id) {
   request_controller_.DeleteRequest(correlation_id);
 }
 
+void VRModule::RegisterService(application_manager::MessagePtr msg) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  commands::Command* command = MobileCommandFactory::CreateCommand(this, msg);
+  if (command) {
+    command->Run();
+    delete command;
+    bool success = true;
+    mobile_apis::Result::eType result = mobile_apis::Result::SUCCESS;
+    std::string info;
+    SendResponseToMobile(msg, success, result, info);
```

Comment:
@bonewell it has no sense to create 2 classes in command for sending 2 same responses. Here we are going to create notifications for HMI by requests from Mobile and send Response. Notifications located in commands